app/components/candidate_interface/carry_over_banner_component.rb

```diff
@@ -17,6 +17,14 @@ def render?
       end
     end
 
+    def references_did_not_come_back_in_time?
+      @application_form.references_did_not_come_back_in_time?
+    end
+
+    def between_cycles?
+      EndOfCycleTimetable.between_cycles_apply_2?
```

Comment:
I think there is also a `between_cycles_apply_1` - should the method name  be more specific?